src/test/java/pageobject_model/page/CloudGooglePricingPage.java

```diff
@@ -0,0 +1,31 @@
+package pageobject_model.page;
+
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import pageobject_model.test.Helpers;
+
+public class CloudGooglePricingPage extends BasePage {
+
+
+    @FindBy(xpath = "//a[@href='https://cloud.google.com/pricing/' and @track-name='seePricing']")
+    private WebElement seePricingLink;
+
+    public CloudGooglePricingPage(WebDriver driver) {
+        super(driver);
+        initElements();
+    }
+
+    public CloudGooglePricingPage openPage() {
+        Helpers.waitUntilClickable("//a[@href='https://cloud.google.com/pricing/' and @track-name='seePricing']", driver);
```

Comment:
Instead of such approach with helpers and transferring driver and locator, you could implement custom instance of WebElement and implement all expected logic there.